### DISCUSSION TITLE
How to use `createShadowRootUi` in unlisted script that is injected using `scripting.executeScript`

### DISCUSSION THREAD

**Discussion by lucadalli** [wxt-dev/wxt#623] [Q&A] [answered]
`createShadowRootUi` expects a `ContentScriptContext` param

**Comment by lucadalli** [comment, marked as answer]
`defineContentScript ` with `registration: 'runtime'`

**Reply by aklinker1** [reply]
Yup, that's correct: https://wxt.dev/api/wxt/interfaces/IsolatedWorldContentScriptDefinition.html#registration
